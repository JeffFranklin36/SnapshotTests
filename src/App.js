import logo from './logo.svg';
import './App.css';
import GitHubCard from './Components/GitHubCard';

function App() {
  return (
    <GitHubCard/>
  );
}

export default App;
