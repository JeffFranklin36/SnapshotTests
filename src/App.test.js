import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer'
import App from './App';

test('renders a snapshot', () => {
  const tree = renderer.create(<App/>).toJSON()
  expect(tree).toMatchSnapshot()
})

