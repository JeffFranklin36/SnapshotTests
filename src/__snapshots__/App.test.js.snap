// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  className="card"
>
  <div
    className="card"
    style={
      Object {
        "width": "18rem",
      }
    }
  >
    <img
      className="card-img-top"
      src="headshot.jpg"
    />
    <div
      className="card-body"
    >
      <div
        className="card-title h5"
      >
        Jeffrey Franklin
      </div>
      <p
        className="card-text"
      >
        Software Developer in Training! Thanks for visiting my page!
      </p>
    </div>
  </div>
</div>
`;
